src/components/HomePage/SearchArea/searchAreaStyle.js

```diff
@@ -82,10 +81,24 @@ export default makeStyles((theme) => ({
         gridColumnStart: 2,
     },
     resetBtnWrapper: {
-        display: "flex",
+        gridRowStart: 4,
+        gridColumnStart: 1,
         "& > *": {
             marginTop: theme.spacing(2),
             marginLeft: theme.spacing(-1),
         },
+        "& .chip-wrapper": {
+            gridColumnStart: 2,
+            gridRowStart: 4,
+        },
+    },
+    showHiddenToggle: {
+        gridRowStart: 4,
+        gridColumnStart: 2,
+    },
+    spaceBtwn: {
```

Comment:
I'm not a big fan of this: this class should have another name, and not one that simply re-states what it does style-wise.